plugins/exporter.koplugin/target/markdown.lua

```diff
@@ -0,0 +1,117 @@
+
+local md = require("template/md")
+local UIManager = require("ui/uimanager")
+local util = require("ffi/util")
+local T = util.template
+local _ = require("gettext")
```

Comment:
order

Reply:
`T` depends on `util` so ordered it like this:
```lua
local md = require("template/md")
local util = require("ffi/util")
local UIManager = require("ui/uimanager")
local T = util.template
local _ = require("gettext")
```

Is it okay now?

Reply:
My bad, I should've linked to the style guide: https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires

Reply:
Is it fine now?

Reply:
Yup, thanks!